vowpalwabbit/active.cc

```diff
@@ -144,32 +144,31 @@ base_learner* active_setup(options_i& options, vw& all)
   bool active_option = false;
   bool simulation = false;
   option_group_definition new_options("Active Learning");
-  new_options
-    .add(make_option("active", active_option).keep().help("enable active learning"))
-    .add(make_option("simulation", simulation).help("active learning simulation mode"))
-    .add(make_option("mellowness",  data->active_c0).default_value(8.f).help("active learning mellowness parameter c_0. Default 8"));
+  new_options.add(make_option("active", active_option).keep().help("enable active learning"))
+      .add(make_option("simulation", simulation).help("active learning simulation mode"))
+      .add(make_option("mellowness", data->active_c0)
+               .default_value(8.f)
```

Comment:
This is still not doing what I expect :-(

How do we bin pack chained function calls?

Reply:
As far as I can tell, this is not possible with clang-format. 

Overall, this patch still seems worthwhile, so I'm going to merge.